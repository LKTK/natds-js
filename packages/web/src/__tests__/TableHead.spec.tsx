import React from 'react';
import * as renderer from 'react-test-renderer';


import {
    TableContainer,
    Table,
    TableRow,
    TableCell,
    TableHead,
} from '..';

describe('TableHead component', () => {
    describe('rendering', () => {
        test('should match to snapshot - TableHead component', () => {
            const component = renderer.create(
                <TableContainer>
                    <Table>
                        <TableHead>
                            <TableRow>
                                <TableCell>Lorem Ipsum dolor</TableCell>
                            </TableRow>
                        </TableHead>
                    </Table>
                </TableContainer>
            );
            expect(component.toJSON).toMatchSnapshot('TableHead component');
        });
    });
});