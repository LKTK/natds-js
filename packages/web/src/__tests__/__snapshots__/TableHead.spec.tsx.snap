// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHead component rendering should match to snapshot - TableHead component: TableHead component 1`] = `[Function]`;
